OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs

```diff
@@ -415,6 +417,10 @@ public static void LoadActors(IniFile file, string section, List<string> players
 
 		public abstract string ParseTreeActor(string input);
 
+		public virtual int ModifyLocationX(string input) { return 0; }
```

Comment:
Can we please change this to a `public virtual CPos ParseActorLocation(int loc)` and include the `mapSize` calculation here?